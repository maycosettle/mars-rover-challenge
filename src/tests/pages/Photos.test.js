import { shallow } from 'enzyme';
import { Photos } from 'pages';

describe('tests on <Photos />', () => {
  const wrapper = shallow(<Photos photos={[]} />);

  test('Should render Photos correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
