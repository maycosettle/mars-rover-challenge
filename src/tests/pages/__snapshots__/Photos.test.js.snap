// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <Photos /> Should render Photos correctly 1`] = `
<Fragment>
  <Header
    title="Mars Rover Challenge"
  />
  <PaperLayout
    className=""
  >
    <PhotoLayout>
      <Filters
        bookmarkOptions={Array []}
        cameraOptions={Array []}
        earthCheckbox={false}
        handleBookmark={[Function]}
        handleCamera={[Function]}
        handleOpen={[Function]}
        handleRover={[Function]}
        handleSearch={[Function]}
        selectedBookmark={Object {}}
        selectedCamera={Object {}}
        selectedRover={Object {}}
        setEarthCheckbox={[Function]}
        setSolCheckbox={[Function]}
        solCheckbox={false}
      />
      <Divider
        className=""
      />
    </PhotoLayout>
    <EmptyTable />
    <Modal
      onClose={[Function]}
      onSubmit={[Function]}
      openModal={false}
    >
      <Bookmark
        onChange={[Function]}
      />
    </Modal>
  </PaperLayout>
</Fragment>
`;
